Doc/library/ssl.rst

```diff
@@ -369,9 +366,9 @@ Certificate handling
    Given the address ``addr`` of an SSL-protected server, as a (*hostname*,
    *port-number*) pair, fetches the server's certificate, and returns it as a
    PEM-encoded string.  If ``ssl_version`` is specified, uses that version of
-   the SSL protocol to attempt to connect to the server.  If ``ca_certs`` is
+   the SSL protocol to attempt to connect to the server.  If *ca_certs* is
    specified, it should be a file containing a list of root certificates, the
-   same format as used for the same parameter in
+   same format as used for the *cafile* parameter in
    :meth:`SSLContext.wrap_socket`.  The call will attempt to validate the
```

Comment:
[`SSLContext.wrap_socket`](https://docs.python.org/3.12/library/ssl.html#ssl.SSLContext.wrap_socket) has no `cafile` parameter, [`SSLContext.load_verify_locations`](https://docs.python.org/3.12/library/ssl.html#ssl.SSLContext.load_verify_locations) has it

Reply:
```suggestion
   same format as used for the *cafile* parameter in
   :meth:`SSLContext.load_verify_locations`.  The call will attempt to validate the
```